vips/process.go

```diff
@@ -300,8 +320,16 @@ func (v *Processor) Process(
 		return imagor.NewBlobFromJsonMarshal(metadata(img, format, stripExif)), nil
 	}
 	format = supportedSaveFormat(format) // convert to supported export format
+	exportParams := &ExportParams{
```

Comment:
Seems creating another ExportParams struct is unnecessary?
Would prefer just populate extra arguments into the `export` function.

The arguments do look a bit more cumbersome I agree, but at least the changes are simpler.

Reply:
My thinking was that there can be many more parameters that someone would like to be able to modify in future, e.g. those in `Jpeg` and `Webp`. Passing them all individually would look very messy.

Reply:
At that point it is probably better to break that down into separated functions, as they have already got their own export params.